### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by aman-bahl** [ScoopInstaller/Scoop#5275] [Help and Questions]
#### Current Behavior

When I'm trying to install to install Scoop, it shows the output in the screenshot given below.

![image](https://user-images.githubusercontent.com/85411047/205678133-c343e56e-b57d-415f-a931-9d284b13c334.png)

### __System details__

**Windows version:** 11

**OS architecture:** 64bit

**PowerShell version:** 7.3.0

**Comment by rashil2000** [comment]
This seems to be an internet connection issue. What's the output of `irm google.com`?

**Reply by aman-bahl** [reply]
Hey there, 
Sorry for my late reply. I get the output as you can see in the screenshot given below.

<img width="960" alt="image" src="https://user-images.githubusercontent.com/85411047/206893303-840efc4f-e47d-4f5d-82a7-feeba7f00621.png">

**Reply by rashil2000** [reply]
And what's the output of `irm get.scoop.sh`?

**Comment by aman-bahl** [comment]
Here you go

<img width="729" alt="image" src="https://user-images.githubusercontent.com/85411047/206897184-a4a52bce-a575-4c46-99ff-7ba74cc0f982.png">

**Reply by rashil2000** [reply]
Are you able to access the same URL through your browser?

**Reply by aman-bahl** [reply]
no, I think I changed my settings in internet options so thats why I'm not able to install scoop.

**Reply by aman-bahl** [reply]
I had changed my TLS settings. So could you just correct me ?? Sorry for taking your time, it was my fault

**Reply by rashil2000** [reply]
I don't follow, what setting did you change?

**Comment by aman-bahl** [comment]
internet options

**Reply by aman-bahl** [reply]
I selected TLS 1.0 and 1.1

**Reply by rashil2000** [reply]
Yeah I think those versions are deprecated.

**Reply by aman-bahl** [reply]
So what do you say I should do now ?

**Reply by aman-bahl** [reply]
Even after disabling both the options Use TLS 1.0 and 1.2 I still cant install scoop

**Reply by rashil2000** [reply]
Try `irm https://raw.githubusercontent.com/scoopinstaller/install/master/install.ps1 | iex`

**Reply by aman-bahl** [reply]
I tried browsing the link as well as using the command which you suggested but both the things didn't work. I get the same error in the terminal

**Reply by rashil2000** [reply]
You need to reset all internet settings to how they were in the default installation of Windows. If you're not able to open the links in the browser too, you need to troubleshoot browser settings too. I'm sorry I can't help you further. The installation script is here: https://github.com/ScoopInstaller/Install/blob/master/install.ps1 

You can run it manually as a last resort.
